src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -4917,6 +4917,12 @@ private void SingleInstanceProcessArgs(string[] args)
 
 		private IRetroAchievements RA { get; set; }
 
+		private void DownloadCoresMenuItemClick(object sender, EventArgs e)
+		{
+			CoreDownloaderForm coreDownloaderForm = new CoreDownloaderForm();

```

Comment:
```suggestion
			using CoreDownloaderForm coreDownloaderForm = new();
```

Reply:
Yeah funny story, `Show()` is non-blocking (versus the blocking `ShowDialog()`), so `using` the form disposes it immediately after it gets shown. (~~and I just realized I did the same with the `EditSubtitlesForm` and never tested it apparently~~)

Reply:
For non-modal dialogs, inherit `IToolForm` and use `Tools.Load`

Reply:
I don't think that's necessary here or in general, the Form gets disposed on close anyway.